cloud/cloud.go

```diff
@@ -196,6 +197,7 @@ func (c *Client) newRequest(ctx context.Context, method string, relativeURL stri
 	if err != nil {
 		return nil, err
 	}
+	req.Header.Add("User-Agent", "terramate/v"+terramate.Version())
```

Comment:
@mariux should we send `terramate/v0.3.1` or `terramate/0.3.1` ?

Reply:
We use `vX.Y.Z` for versioning as well, so I think this should be done consistently here?